subsystems/mvdm/ntvdm/dos/dos32krnl/process.c

```diff
@@ -796,10 +796,45 @@ WORD DosCreateProcess(IN LPCSTR ProgramName,
         /* Those are handled by NTVDM */
         case SCS_WOW_BINARY:
         {
-            DisplayMessage(L"Trying to load '%S'.\n"
-                           L"WOW16 applications are not supported internally by NTVDM at the moment.\n"
-                           L"Press 'OK' to continue.",
-                           ProgramName);
+            static const PCSTR AppName = "\"%ProgramFiles%\\otvdm\\otvdmw.exe\" ";
```

Comment:
According to the following @learn-more's comment: https://jira.reactos.org/browse/CORE-17852?focusedCommentId=130726&page=com.atlassian.jira.plugin.system.issuetabpanels:comment-tabpanel#comment-130726, I don't know where exactly, but otvdm seems to write a path to its exe in registry after installing settings via inf file. This current location will obviously not work if otvdm is extracted in different location than Program Files. Maybe it makes sense to retrieve this path from registry instead of just hardcoding Program Files? :thinking: And add a note in `DisplayMessage` like "to use the integration, you need to install otvdm's registry setting from install.inf file"?

Reply:
Here it is:
![otvdm-registry-path](https://user-images.githubusercontent.com/26385117/142764539-f79c4599-9eb3-4492-bfe0-37b352931512.png)

Although we also should to fix the infs first, to make them installable on x86.

Reply:
@oleg-dubinskiy, In all of my testing I never installed anything using the '*.inf' files. Just copying these files into a known location and linking to either the 'otvdm.exe' or 'otvdmw.exe' worked fine for me. Can you provide any explanation about a benefit of using the '*.inf' install files? This function seems to be a way to get this integrated into a normal windows environment where we do not have the ability to customize the OS like we have in ReactOS. Thank you for you comments. Doug Lyons

Reply:
Well, the only benefit of using infs is, otvdm can be launched from **any location** to which it was extracted, besides Program Files. Otherwise, user may not know about this, and extract otvdm into any other path that he will decided better. And then, he may be confused, because otvdm will not launch (obviously because it doesn't exist in Program Files).
OK, if the method above isn't an option, then we can just add a note in `DisplayMessage` call about it should be extracted only in Program Files. Then user will understand this from the start, and then he will extract otvdm in the correct location. **In this case, we don't need to use inf files at all.** :slightly_smiling_face: 
Regarding other aspects, I'm completely agreed that using infs has no any another benefit besides described above.

Reply:
@oleg-dubinskiy, I have added the note to the ```DisplayMessage`` about installing into "C:\Program Files" as you suggested. Thanks. Doug Lyons